cluster-autoscaler/cloudprovider/aws/aws_manager_test.go

```diff
@@ -48,6 +46,9 @@ func resetAWSRegion(value string, present bool) {
 	}
 }
 
+/*
```

Comment:
Please create an issue for AWS cloudprovider about this.

Reply:
Will do after merging. Adding to followups